libtransmission/clients.c

```diff
@@ -117,12 +117,12 @@ decodeBitCometClient (char * buf, size_t buflen, const uint8_t * id)
     const char * name;
     const char * mod = NULL;
 
-    if (memcmp (id, "exbc", 4) == 0) mod = "";
-    else if (memcmp (id, "FUTB", 4) == 0) mod = " (Solidox Mod) ";
-    else if (memcmp (id, "xUTB", 4) == 0) mod = " (Mod 2) ";
+    if (strncmp (id, "exbc", 4) == 0) mod = "";
```

Comment:
Changes in this file lead to lots of warnings, as `id` is not a pointer to `char`, but `uint8_t`: "pointer targets in passing argument 1 of ‘strncmp’ differ in signedness [-Wpointer-sign]". My machine currently has GCC 5.4.0 installed (in case you don't see those warnings with your compiler).